ui-components/src/blocks/InfoCard.tsx

```diff
@@ -18,13 +18,13 @@ const InfoCard = (props: InfoCardProps) => {
   return (
     <div className={wrapperClasses.join(" ")}>
       {props.externalHref ? (
-        <h4 className="info-card__title">
+        <h3 className="info-card__title">
```

Comment:
I updated this because of the rule "Ensures the order of headings is semantically correct". We had a jump from an h2 at the InfoCard group level to this h3. It doesn't affect the styling, just the semantics

[More info](https://dequeuniversity.com/rules/axe/4.2/heading-order?application=axe-puppetee)

Reply:
Longterm plan, should have props for tags so that you set it at the appropriate level depending on the context. Add a TODO to add props, or create an issue about it.

Reply:
Added #1516 to address this.